### PR TITLE
Adding docs for PaddleNLP

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

**Comment by sijunhe** [comment]
hi @osanseviero . I think this is ready for review

File: docs/hub/paddlenlp.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet1-dark.png"/>
+</div>
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet2.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet2-dark.png"/>
+</div>
+
+## Sharing your models
+
+You can share your `PaddleNLP` models by using the `save_to_hf_hub` method under all `Model` and `Tokenizer` class.
```
Comment (osanseviero) [inline docs/hub/paddlenlp.md:55]:
```suggestion
You can share your `PaddleNLP` models by using the `save_to_hf_hub` method under all `Model` and `Tokenizer` classes.
```

File: docs/hub/paddlenlp.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+tokenizer.save_to_hf_hub(repo_id="<my_org_name>/<my_repo_name>")
+model.save_to_hf_hub(repo_id="<my_org_name>/<my_repo_name>")
+```
+
+
+## Additional resources
+
+* PaddlePaddle Installation [guide](https://www.paddlepaddle.org.cn/en/install).
+* PaddleNLP [GitHub Repo](https://github.com/PaddlePaddle/PaddleNLP).
+* [PaddlePaddle on Huggingface Hub](https://huggingface.co/PaddlePaddle)
```
Comment (osanseviero) [inline docs/hub/paddlenlp.md:72]:
```suggestion
* [PaddlePaddle on the Hugging Face Hub](https://huggingface.co/PaddlePaddle)
```

**Comment by sijunhe** [comment]
@osanseviero addressed your comments

**Comment by osanseviero** [comment]
@radames would you be up for taking a look into the images?

File: docs/hub/paddlenlp.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+If you want to see how to load a specific model, you can click `Use in paddlenlp` and you will be given a working snippet that you can load it! 
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet1.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet1-dark.png"/>
+</div>
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet2.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-fastai_snippet2-dark.png"/>
```
[the remark this answers is not in the document]
Comment (radames) [reply docs/hub/paddlenlp.md:50]: hi @sijunhe ,  sorry for the delay, what help do you need with the images? they're already hosted on our dataset 👍 do you need help with layout?

**Comment by radames** [comment]
I've added the images please check @osanseviero @sijunhe thanks

**Review by sijunhe — COMMENTED** [review]
LGTM!

**Review by osanseviero — APPROVED** [review]
Thank you! LGTM!

**Pull request merged**